model/service.go

```diff
@@ -44,15 +45,28 @@ type Endpoint struct {
 	// Address of the endpoint, typically an IP address
 	Address string `json:"ip_address,omitempty"`
 	// Port on the host address
-	Port int `json:"port,omitempty"`
+	Port int `json:"port"`
+	// Name of the port classifies ports for a single service
+	Name string `json:"name,omitempty"`
```

Comment:
What are we using the name field for, given that we have HTTP being designated as a protocol by itself.